backend/api/events/types.py

```diff
@@ -1,20 +1,18 @@
-from typing import TYPE_CHECKING, Optional
+from strawberry import LazyType
+from typing import Optional
 
 import strawberry
-from strawberry.types.datetime import DateTime
+from strawberry.schema.types.base_scalars import DateTime
 
 from ..helpers.i18n import make_localized_resolver
 from ..helpers.images import resolve_image
 from ..helpers.maps import Map, resolve_map
 
-if TYPE_CHECKING:  # pragma: no cover
-    from api.conferences.types import Conference
-
 
 @strawberry.type
 class Event:
     id: strawberry.ID
-    conference: "Conference"
+    conference: LazyType["Conference", "api.conferences.types"]
```

Comment:
can you try with:

```suggestion
    conference: "api.conferences.types.Conference"
```

just curious to see if this works, maybe LazyType is actually not needed :)

Reply:
it works but I need to `import api` at the top, I am not sure I like it 🤔

Reply:
you can hide it under `if TYPE_CHECKING`, anyway both solutions are ok for now :)